scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+[id="cnf-talm-for-cluster-updates"]
+= {cgu-operator-full} for cluster updates
+include::_attributes/common-attributes.adoc[]
+:context: cnf-topology-aware-lifecycle-manager
+
+toc::[]
+
+You can use the {cgu-operator-first} to manage the software lifecycle of multiple single-node OpenShift clusters. {cgu-operator} uses {rh-rhacm-first} policies to perform changes on the target clusters.
+
+:FeatureName: {cgu-operator-full}
+include::snippets/technology-preview.adoc[]
+
+include::modules/cnf-about-topology-aware-lifecycle-manager-config.adoc[leveloffset=+1]
+
+include::modules/cnf-about-topology-aware-lifecycle-manager-policies.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-manager-installation-web-console.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc[leveloffset=+1]
+
+include::modules/cnf-about-topology-aware-lifecycle-manager-blocking-crs.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-policies-concept.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+For more information about `PolicyGenTemplate` CRD, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-the-policygentemplate_ztp-deploying-disconnected[About the PolicyGenTemplate].
+
+include::modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-backup-concept.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-precache-concept.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-manager-precache-feature.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about troubleshooting, see xref:../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Container Platform Troubleshooting Operator Issues].
+
+* For more information about using {cgu-operator-full} in the ZTP workflow, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#cnf-topology-aware-lifecycle-manager[Updating managed policies with {cgu-operator}].
```

Comment:
Additional resources are just supposed to be bare links now (it's a Jupiter thing):
```suggestion
* xref:../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Container Platform Troubleshooting Operator Issues].

* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#cnf-topology-aware-lifecycle-manager[Updating managed policies with {cgu-operator}].
```